File: SCC0600/exercíciosPy/ex39.py
n1 = int(input())
n2 = str(input())

def numerize (arr, rang):
  newarr = [0] * rang
  splited = arr.split(' ')
  for k in range(rang):
    newarr[k] = int(splited[k])
  return newarr

numerizado = numerize(n2, n1)
numerizado.sort(reverse=True)
normal = numerize(n2, n1)
maior = numerizado[0]
res = normal.index(numerizado[0])

print(maior)
print(res)

File: SCC0600/exerciciosC/ex39.c
#include<stdio.h>

int main (void) {
    int size;
    scanf("%d", &size);
    int arr[size];
    for (int i = 0; i < size; i++) {
        scanf("%d", &arr[i]);
    }
    int pos = 0;
    int maior = arr[0];
    for (int i = 0; i < size; i++) {
        if (arr[i] > maior) {
            pos = i;
            maior = arr[i];
        }
    }
    printf("%d\n%d", maior, pos);
}